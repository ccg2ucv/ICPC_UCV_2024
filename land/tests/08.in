1000000 1000000 2000000 1000000 1000000 2000000 2000000 2000000
1500000 1500000 2500000 1500000 1500000 2500000 2500000 2500000
