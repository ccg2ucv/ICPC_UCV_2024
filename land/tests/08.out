YES 250000000000 1000000000000 1000000000000
